loopy/transform/iname.py

```diff
@@ -1737,6 +1736,49 @@ def add_inames_to_insn(kernel, inames, insn_match):
 # }}}
 
 
+# {{{ remove_inames_from_insn
+
+@for_each_kernel
+def remove_inames_from_insn(kernel, inames, insn_match):
```

Comment:
Type-annotate?

Reply:
Done in eea9df173e675c7caee7d8f539123293088fbe9a